src/gpuarray/config.h

```diff
@@ -12,19 +12,15 @@
   #else
    #define GPUARRAY_PUBLIC __declspec(dllimport)
   #endif
-  #define GPUARRAY_LOCAL
```

Comment:
Should this commit and the previous should be included in 0.6? If so, make a new PR for this.